src/SearchForm.tsx

```diff
@@ -75,8 +76,7 @@ export const SearchForm = ({
   submit: (fullSearch: SearchProps) => void;
 }) => {
   const [newSearch, setNewSearch] = useState("");
-  const [newMediaType, setNewMediaType] = useState("movie");
-
+  const [newMediaType, setNewMediaType] = useState<MediaType>("movie");
```

Comment:
What's the decision made to not put this state in the reducer?

Reply:
i think that is in the reducer right? i might have just forgotten to delete it

Reply:
Ah yes I see it